ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -208,31 +208,31 @@ export class ReposList extends React.Component<
                     url: !sshValues.url && 'Repository URL is required'
                 };
             case ConnectionMethod.HTTPS:
-                const httpsValues = params as NewHTTPSRepoParams;
+                const validURLValues = params as NewHTTPSRepoParams;
                 return {
                     url:
-                        (!httpsValues.url && 'Repository URL is required') ||
-                        (this.credsTemplate && !this.isHTTPSUrl(httpsValues.url) && !httpsValues.enableOCI && 'Not a valid HTTPS URL'),
-                    name: httpsValues.type === 'helm' && !httpsValues.name && 'Name is required',
-                    username: !httpsValues.username && httpsValues.password && 'Username is required if password is given.',
-                    password: !httpsValues.password && httpsValues.username && 'Password is required if username is given.',
-                    bearerToken:
-                        (httpsValues.password && httpsValues.bearerToken && 'Either the password or the bearer token must be set, but not both.') ||
-                        (httpsValues.type != 'git' && 'Bearer token is only supported for Git BitBucket Data Center repositories'),
```

Comment:
I don't think you meant to remove this. We don't want to drop the bearer token support.